docs/man/sat-jobstat.8.rst

```diff
@@ -0,0 +1,60 @@
+=============
+ SAT-JOBSTAT
+=============
+
+-------------------------------------------------------
```

Comment:
nit: now that line 6 has been shortened could you shorten these lines to match as well? (This line and line 7).